packages/react-codemirror/src/e2e_tests/configuration.spec.tsx

```diff
@@ -1,8 +1,6 @@
 import { expect, test } from '@playwright/experimental-ct-react';
 import { CypherEditor } from '../CypherEditor';
 
-test.use({ viewport: { width: 500, height: 500 } });
```

Comment:
This cleanup is so that things shape up in the same way that they do in the codemirror demo